tasks/config/databases.yml

```diff
@@ -4,38 +4,42 @@
   command: "clickhouse-client -u {{ clickhouse_admin_user }} --password {{ clickhouse_admin_password }} -q 'CREATE DATABASE IF NOT EXISTS `{{ clickhouse_default_database }}`'" # noqa 204
   changed_when: False
   when: clickhouse_default_database is defined
-  no_log: False
+  no_log: True
```

Comment:
As it can log sensible information (user/password) no_log must be maintained, I think. Same for rest of similar changes.